### DISCUSSION TITLE
backbone webpack build warning: export default was not found

### DISCUSSION THREAD

**Discussion by ohmyvander** [babel/babel#15005] [Q&A] [answered]
**What is the current behavior?**

I have a project using backbone. When I built the project in my linux server, everything was ok. But when I built it in a docker container, then something went wrong. All node version and npm version are same in two environments, and the operating system are similar too. Here is a very simple example: [webpack-build-demo](https://github.com/ohmyvander/webpack-build-demo)

**If the current behavior is a bug, please provide the steps to reproduce.**

After running `npm install`, when I ran `npm run build` in my linux server, the result was like that:
![image](https://user-images.githubusercontent.com/8723745/193414679-0a7d299f-4395-4352-a216-1dbd6fb35ae8.png)
And if I run the bundle js file, the browser prints correct result:
![image](https://user-images.githubusercontent.com/8723745/193414916-090bb8e3-a093-4bc9-939b-3e19c7699faf.png)
But if I do the same thing in my docker container, there is a different result:
![image](https://user-images.githubusercontent.com/8723745/193415201-a7e0de54-8a30-4018-a3e0-09d0f4893c54.png)
And the browser prints error message:
![image](https://user-images.githubusercontent.com/8723745/193415223-676524b4-2eda-4b53-808e-8c521fac5bc9.png)

**Other relevant information:**
webpack version:  4.46.0
Node.js version: 12.16.2

**Comment by liuxingbaoyu** [comment, marked as answer]
Bundling is handled by webpack, and babel should be platform agnostic in transpiling code, so this is more likely a webpack related issue, and I suggest you ask them.
